### PR TITLE
NodeDescription tag Localization

### PR DESCRIPTION
**Pull request by zora-wang**
Hi @Benglin , NodeDescriptions have been localized. please check them 
Thanks

### CODE REVIEW

**Comment by zora-wang** [comment]
Hi  @Benglin . please merge the files again. I have changed them.. Thanks

File: src/DynamoCore/Nodes/DummyNode.cs [hunk 1]
```diff
@@ -5,11 +5,12 @@
 using System.Xml;
 using Autodesk.DesignScript.Runtime;
 using Dynamo.Models;
+using Dynamo.Properties;
 
 namespace DSCoreNodesUI
 {
     [NodeName("Legacy Node")]
-    [NodeDescription("This is an obsolete node")]
+    [NodeDescription("DummyNode", typeof(Resources))]
```
Comment (Benglin) [inline src/DynamoCore/Nodes/DummyNode.cs]: Hi @zora-wang, the term `DummyNode` here isn't very descriptive. Localization team would look at this and not sure what to do with it, but if we name it as `DummyNodeNodeDescription`, then it becomes clear, it is *a description for the dummy node*. Does that make sense? So please go through all the `NodeDescription` tags in the solution, and then make sure they all follow this format: `NodeXyzNodeDescription`.

Thanks!

File: src/DynamoCore/Nodes/ZeroTouch/DSFunction.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@ namespace Dynamo.Nodes
     ///     DesignScript function node. All functions from DesignScript share the
     ///     same function node but internally have different procedure.
     /// </summary>
-    [NodeName("Function Node"), NodeDescription("DesignScript Builtin Functions"),
+    [NodeName("Function Node"), NodeDescription("DSFunctionBuiltin",typeof(Resources)),
```
Comment (Benglin) [inline src/DynamoCore/Nodes/ZeroTouch/DSFunction.cs]: For this, it will be `DSFunctionNodeDescription`.

**Comment by zora-wang** [comment]
noticed. I will correct them

Thanks :)

File: src/DynamoCoreWpf/Properties/Resources.en-US.resx [hunk 3]
```diff
@@ -1519,4 +1519,25 @@ Click CANCEL to cancel the update.</value>
...
+  </data>
+  <data name="Point3Node" xml:space="preserve">
+    <value>Create a point in 3D space.</value>
+  </data>
+  <data name="OutputNodeDescription" xml:space="preserve">
+    <value>A function output, use with custom nodes</value>
+  </data>
+  <data name="SymbolNodeDescription" xml:space="preserve">
+    <value>A function parameter, use with custom nodes</value>
+  </data>
```
Comment (Benglin) [inline src/DynamoCoreWpf/Properties/Resources.en-US.resx]: `SymbolNodeDescription` being used twice, but there are differences in the original string contents, do use the longer version.

File: src/DynamoCore/Properties/Resources.resx [hunk 4]
```diff
@@ -503,7 +502,7 @@ If you don't mind, it would be helpful for you to send us your file. That will m
   </data>
   <data name="ToolTipSymbol" xml:space="preserve">
     <value>Symbol</value>
-	</data>
+  </data>
```
Comment (Benglin) [inline src/DynamoCore/Properties/Resources.resx]: `Resources.resx` should be exactly the same as `Resources.en-US.resx`, can you copy `Resources.en-US.resx` and overwrite `Resources.resx` since `Resources.resx` has some empty line changes?

File: src/DynamoCore/Nodes/Custom Nodes/Function.cs [hunk 5]
```diff
@@ -360,7 +361,7 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
 
     [NodeName("Output")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("A function output, use with custom nodes")]
+    [NodeDescription("FunctionOutput",typeof(Resources))]
```
Comment (Benglin) [inline src/DynamoCore/Nodes/Custom Nodes/Function.cs]: `FunctionOutputNodeDescription`?

File: src/DynamoCoreWpf/Nodes/DSFunction.cs [hunk 6]
```diff
@@ -417,7 +417,7 @@ internal override IEnumerable<AssociativeNode> BuildAst(List<AssociativeNode> in
     /// DS Function Node that support Variable Arguments.
     /// </summary>
     [NodeName("Function Node w/ VarArgs")]
-    [NodeDescription("DesignScript Builtin Functions")]
+    [NodeDescription("DSVarArgFuncBuiltin",typeof(Resources))]
```
Comment (Benglin) [inline src/DynamoCoreWpf/Nodes/DSFunction.cs]: `DSVarArgFunctionNodeDescription`?

File: src/DynamoCoreWpf/Properties/Resources.en-US.resx [hunk 7]
```diff
@@ -1519,4 +1519,25 @@ Click CANCEL to cancel the update.</value>
...
+  <data name="CustomNodeDescription" xml:space="preserve">
+    <value>Instance of a Custom Node</value>
+  </data>
+  <data name="Point2Node" xml:space="preserve">
+    <value>Create a point in the XY plane.</value>
+  </data>
+  <data name="Point3Node" xml:space="preserve">
+    <value>Create a point in 3D space.</value>
+  </data>
```
Comment (Benglin) [inline src/DynamoCoreWpf/Properties/Resources.en-US.resx]: Where do we get `Point2Node` and `Point3Node` from? Do remove them if we don't need these.

File: src/Libraries/CoreNodeModels/Input/BaseTypes.cs [hunk 8]
```diff
@@ -14,12 +14,12 @@
...
 using ProtoCore.Namespace;
 using String = DSCoreNodesUI.String;
-
+using DSCoreNodesUI.Properties;
 namespace Dynamo.Nodes
 {
     [NodeName("String")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("Creates a string.")]
+    [NodeDescription("StringNodeDescription",typeof(Resources))]
```
Comment (Benglin) [inline src/Libraries/CoreNodeModels/Input/BaseTypes.cs]: Is there a reason we don't name this `StringInputNodeDescription`?

File: src/Libraries/CoreNodeModels/Input/BaseTypes.cs [hunk 9]
```diff
@@ -118,7 +118,7 @@ where attr.Name.Equals("value")
 
     [NodeName("Number")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("Creates a number.")]
+    [NodeDescription("NumberNodeDescription",typeof(Resources))]
```
Comment (Benglin) [inline src/Libraries/CoreNodeModels/Input/BaseTypes.cs]: Is there a reason we don't name this `DoubleInputNodeDescription`?

File: src/Libraries/CoreNodeModels/Input/BoolSelector.cs [hunk 10]
```diff
@@ -37,7 +37,8 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
     [NodeName("Boolean")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
     [NodeDescription("BooleanDescription", typeof(Properties.Resources))]
-    [NodeSearchTags("BooleanSelectorSearchTags", typeof(Properties.Resources))]
+    [NodeSearchTags("BooleanSelectorSearchTags", 
+typeof(Properties.Resources))]
```
Comment (Benglin) [inline src/Libraries/CoreNodeModels/Input/BoolSelector.cs]: Why is this file changed? there are no code changes, do we really need `using DSCoreNodesUI.Properties`?

File: src/Libraries/CoreNodeModels/Input/Sliders.cs [hunk 11]
```diff
@@ -295,7 +295,7 @@ protected override bool ShouldDisplayPreviewCore()
 
     [NodeName("Integer Slider")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("A slider that produces integer values.")]
+    [NodeDescription("IntSliderNodeDescription",typeof(Resources))]
```
Comment (Benglin) [inline src/Libraries/CoreNodeModels/Input/Sliders.cs]: Please follow the pattern: `ClassNameNodeDescription` (and replace the `ClassName` with the actual class name), can we redo all these `NodeDescription` thing?
